app/routes/register.tsx

```diff
@@ -141,9 +142,14 @@ export const loader: LoaderFunction = async ({ request }) => {
       oneClickUuid
     );
     if (result) {
+      const firstName = result?.credentials?.fullName?.firstName;
+
+      // Because the user has canceled 1-click flow, the credentials was not shared, so we need to logout the user.
```

Comment:
```suggestion
      // Because the user has canceled 1-click flow, the credentials was not shared, so we need to logout the user.
      // Customer Note: in a real implementation this ought to fall back to the standard sign up form.
```

Considering this project is for reference, we should note what the customer implementation should do. I also think we probably should start using a convention of something like `Customer Note`, which we can reference in the README, to call out these interesting points in the source code.

Reply:
I liked that customer note convention!